packages/core-components/src/components/LogViewer/LogLine.tsx

```diff
@@ -137,6 +139,21 @@ export function calculateHighlightedChunks(
   return chunks;
 }
 
+const renderLink = ({
+  attributes,
+  content,
+}: {
+  attributes: { [attr: string]: any };
```

Comment:
I know we don't like to use `any` but this was the best match I could find from Linkify. If there's a better option I'll gladly change it